internal/stacks/stackruntime/internal/stackeval/component.go

```diff
@@ -266,9 +268,8 @@ func (c *Component) PlanIsComplete(ctx context.Context) bool {
 			// get returned by a different return path.
 			return false
 		}
-		if !plan.Complete {
-			return false
-		}
+
+		return plan.Complete
```

Comment:
I think we don't want this change? since it will only look at the first instance and not inspect the rest?

Reply:
Oh yeah, I misread the parentheses on this one and thought that's a weird way to write it :D